packages/css-framework/src/components/_alert.scss

```diff
@@ -1,48 +1,52 @@
+@use "sass:map";
+@use "../abstracts/mixins" as m;
+@use "../abstracts/functions" as f;
```

Comment:
Do we have to expose it via this aliased object (`f`)? Can we no longer just import and use `color()` if we use modules?

Reply:
We can easily import functions or mixins as `@use "../abstracts/mixins" as *;`, where the * adds the function to the global namespace. This'll then let you call `color()` without needing the namespace.

I decided to namespace them with `f` & `m` though as if we decide to import a different external project, it'll mean we won't have clashes.